pkg/install/v1alpha1/cstor_pool.go

```diff
@@ -44,6 +44,13 @@ spec:
   # nothing exists at the given path i.e. an empty directory will be created.
   - name: HostPathType
     value: DirectoryOrCreate
+  # PersistentStoragePath is base directory to store lock, sock and cache and core files.
```

Comment:
is this comment correct?

Reply:
I will update it I picked it from different PR.